### ISSUE TITLE
How to enable JSON mode for azure-openai ?

### ISSUE THREAD

**Issue by Plawn** [quarkiverse/quarkus-langchain4j#466]
Hello, 

Sorry to bother you with such questions :), 

I can't seem to enable the json mode with azure-open, looking in the code, it seems it should work with this configuration but It's not working.

```yaml
quarkus:
  # settings for the models
  langchain4j:
    azure-openai: # should be azure open ai
      api-key:<key>
      resource-name: <name>
      deployment-name: <name>
      response-format: json_object
```

Looking in the langchain4j documentation, It looks like the way to configure the `responseFormat` is not the Same for the OpenAIClient and the AzureOpenAiClient.

Thanks

**Comment by UnvirtualHH** [comment]
It looks like the responseFormat is just implemented for the image model, actually not for chat model. Maybe I can provide a PR tomorrow.

**Comment by Plawn** [comment]
Many are the thanks

**Comment by geoand** [comment]
> Looking in the langchain4j documentation

Can you please point to where you see this in the documentation?

**Comment by Plawn** [comment]
Here it is ! 

https://docs.langchain4j.dev/tutorials/ai-services/#json-mode

🚀

**Comment by geoand** [comment]
Oh I see!

So indeed we need to add that AzureAI feature here too.

**Comment by geoand** [comment]
> It looks like the responseFormat is just implemented for the image model, actually not for chat model. Maybe I can provide a PR tomorrow.

This would be very nice!

**Comment by UnvirtualHH** [comment]
I'm still struggling with formatting lf/crlf topics, fix is locally working, but not sure what is wrong with my ide...

**Comment by geoand** [comment]
You can just do `mvn install -f openai/azure-openai` and commit after the changes the auto-formatting makes.

**Comment by UnvirtualHH** [comment]
Well, looks like I got it. Here is the PR: https://github.com/quarkiverse/quarkus-langchain4j/pull/483

**Comment by geoand** [comment]
Closed by #483
